pruntime/enclave/src/contracts/balance.rs

```diff
@@ -56,8 +59,9 @@ pub enum Request {
 pub struct TransferData {
     dest: AccountIdWrapper,
     amount: chain::Balance,
-    signature: Option<Vec<u8>>,
+    signature: Vec<u8>,
```

Comment:
To be honest I'd prefer to have the signature separated from the other fields like below. The reason is that the signature is a commitment to `TransferData`, rather than a part of its content. The separation also make future change to the inner data easier. But before it's too complicated I'm fine with the current approach.

```rust
struct TransferData {
    dest: AccountIdWrapper,
    amount: chain::Balance,
    sequence: SequenceType,
}
struct SignedTransferData {
    transfer_data: TransferData,
    signature: Vec<u8>,
}
```

Reply:
I'll refactor later